shuup/admin/static_src/base/js/tour.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * This file is part of Shuup.
+ *
+ * Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+(function($) {
```

Comment:
This code is used for all tours. When "re-running" a tour, we run the tour for the page + the chrome.